documentation/sphinx/source/api-python.rst

```diff
@@ -899,6 +899,13 @@ Transaction options
 
 .. _api-python-future:
 
+Transaction misc functions
+--------------------------
+
+.. method:: Transaction.get_estimated_range_size_bytes(begin_key, end_key)
+
+    Get the estimated byte size of the given key range. Returns a :class:`FutureInt64`.
```

Comment:
If I'm not crazy, it seems like we're missing documentation changes for the getApproximateSize API that would have changed `FutureVersion` to `FutureInt64`, so this link won't refer to anything in this documentation. We should go ahead and make that update too if we're going to refer to it.

Also, this block shouldn't exist under the `api-python-future` link, which is meant to apply to the section below. Probably this section should live above the options section, since the options go from documenting `Transaction` to documenting `Transaction.options`.

Reply:
Done. Changed the `FutureVersion` to `FutureInt64` and some text below it.